test/model/content_test.dart

```diff
@@ -395,6 +395,23 @@ class ContentExample {
       ParagraphNode(links: null, nodes: [TextNode("some content")]),
     ]);
 
+  static const codeBlockNodesWithMultipleClasses = ContentExample(
+    'code block nodes with multiple class names',
+    '```yaml\n- item\n```',
+    expectedText: '- item',
+    // https://chat.zulip.org/#narrow/channel/7-test-here/topic/Greg/near/1949014
+    '<div class="codehilite" data-code-language="YAML">'
+      '<pre><span></span><code><span class="p p-Indicator">-</span>'
+      '<span class="w"> </span>'
+      '<span class="l l-Scalar l-Scalar-Plain">item</span>\n'
+      '</code></pre></div>', [
+      CodeBlockNode([
```

Comment:
super nit: indent the string continuations deeper so they don't look like part of the same series as the list element below

```suggestion
    '<div class="codehilite" data-code-language="YAML">'
        '<pre><span></span><code><span class="p p-Indicator">-</span>'
        '<span class="w"> </span>'
        '<span class="l l-Scalar l-Scalar-Plain">item</span>\n'
        '</code></pre></div>', [
      CodeBlockNode([
```

This is similar to what we do with `if` conditions. You can also see it in action on the similar examples codeBlockHighlightedShort and codeBlockHighlightedMultiline above.